spec/asyncapi.md

```diff
@@ -918,6 +940,47 @@ bindings:
 
 
 
+#### <a name="operationReplyObject"></a>Operation Reply Object
+
+Describes the reply part that MAY be applied to an Operation Object. If an operation implements the request/reply pattern, the reply object represents the response message.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="operationReplyObjectAddress"></a>address | [Operation Reply Address Object](#operationReplyAddressObject) &#124; [Reference Object](#referenceObject) | Definition of the address that implementations MUST use for the reply.
```

Comment:
If we add possibility to use the `$ref` for `address` field, we should also be able to define the reusable reply addresses in components -> `components.replyAddresses`.

cc @GreenRover @fmvilas @derberg

Reply:
ok changed it. ok?

Reply:
Agree. Left a suggestion below @GreenRover.